### PR TITLE
change naming convention for Bootstrap service-name

### PR DESCRIPTION
**Pull request by patriknw**
Corresponding to what the operator will use.

### CODE REVIEW

File: docs-source/docs/modules/microservices-tutorial/examples/00-shopping-order-service-scala/src/main/resources/local-shared.conf [hunk 1]
```diff
@@ -18,8 +18,7 @@ akka.management.cluster.bootstrap.contact-point-discovery {
 }
 
 akka.discovery.config.services {
-  // service name == lower case ActorSystem name by default
-  "shoppingorderservice" {
+  "shopping-order-service" {
```
Comment (ignasi35) [inline docs-source/docs/modules/microservices-tutorial/examples/00-shopping-order-service-scala/src/main/resources/local-shared.conf:21]: Maybe we can keep the comment with the recommended convention?

Comment (ignasi35) [reply docs-source/docs/modules/microservices-tutorial/examples/00-shopping-order-service-scala/src/main/resources/local-shared.conf:21]:
```suggestion
   // (recommended) service name == kebab case ActorSystem name
   "shopping-order-service" {
```

**Review by ignasi35 — APPROVED** [review]
LGTM

File: docs-source/docs/modules/microservices-tutorial/examples/00-shopping-order-service-scala/src/main/resources/local-shared.conf (hunk 1 above)
Comment (patriknw) [reply docs-source/docs/modules/microservices-tutorial/examples/00-shopping-order-service-scala/src/main/resources/local-shared.conf:21]: When we specify the explicit `service-name` it has no connection with the ActorSystem name anymore, so I don't think we need to confuse the user with any additional recommendation. When we generate the template the user will only specify "shopping-order-service" and we will generate the ActorSystem name with those conventions.

**Comment by patriknw** [comment]
I'll look into the test failures, could be some real mistake

**Review by ignasi35 — APPROVED** [review]

**Pull request merged**